lib/super_good/solidus_taxjar/api.rb

```diff
@@ -36,10 +36,18 @@ def tax_rates_for(address)
       end
 
       def create_transaction_for(order)
-        transaction_id = TransactionIdGenerator.next_transaction_id(order: order)
+        latest_transaction_id =
+          OrderTransaction.latest_for(order)&.transaction_id
```

Comment:
That sounds reasonable to me.

Reply:
@forkata I kind of want to put this off into another PR, just because this PR is adding a lot as it is and I think we'd want to introduce new tests if we change the `next_transaction_id` implementation.

I will capture this in a new ticket if that works for you.

Reply:
Captured in #168

Reply:
Thanks @benjaminwil!